docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -66,67 +80,17 @@ class Sidebar extends React.Component<any, any> {
     if (!hasModifier && isAZ && bodyHasFocus) this._searchInput.focus()
```

Comment:
```suggestion
    if (!hasModifier && isAZ && bodyHasFocus) this._searchInput.current.focus()
```

https://github.com/stardust-ui/react/pull/1682/files#r307795621